libs/designsystem/item/src/item.component.scss

```
@@ -94,6 +94,15 @@
 .disclosure {
   padding-left: #{utils.size('xxxs')};
   color: #{utils.get-color('semi-dark')};
+  line-height: 0;
```

Consider changing to:
```suggestion
  display: inline-block;
  line-height: 1;
```